File: Smile Base/src/main/java/de/vd40xu/smilebase/model/emuns/AppointmentType.java
package de.vd40xu.smilebase.model.emuns;

public enum AppointmentType {
    QUICKCHECK, EXTENSIVE, SURGERY;

    public long getDuration() {
        return switch (this) {
            case QUICKCHECK -> 30;
            case EXTENSIVE -> 60;
            case SURGERY -> 120;
        };
    }
}

File: Happy Face/src/models/enums/AppointmentType.ts
export enum AppointmentType {
    QUICKCHECK = 'QUICKCHECK',
    EXTENSIVE = 'EXTENSIVE',
    SURGERY = 'SURGERY'
}

export const appointmentDurations: Record<AppointmentType, number> = {
    [AppointmentType.QUICKCHECK]: 30,
    [AppointmentType.EXTENSIVE]: 60,
    [AppointmentType.SURGERY]: 120,
};